linux-gl-js/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:16.04
```

Comment:
Can you please derive this image from one of our other images? See https://github.com/mapbox/mbgl-ci-images/blob/master/README.md for more information on the reasoning behind that. I think `linux` would be a good choice. It contains a compiler too, so I think the only thing missing is `yarn` + the gl-js checkout. In the future, we could add `yarn` to the base image as well, since it's a minimal dependency and could come in handy for other projects as well.

To derive them from the current set of images, change `FROM mbgl/ci:latest-linux` to `FROM mbgl/ci:r4-linux` locally, build the image, then commit the version with "latest".